src/data/huskelapp/huskelappTypes.ts

```diff
@@ -13,13 +13,15 @@ const texts = {
 
 export interface HuskelappRequestDTO {
   oppfolgingsgrunn: Oppfolgingsgrunn;
+  frist: string | null;
```

Comment:
Vi bruker `string` for dato både på avvent-frist og andre steder. Jeg hadde `Date` først, men slet veldig med konverteringer frem og tilbake i testen. Feks tok den dagen idag til å være 2023-11-30T23:00:00, så ble mye rar logikk for å det til å være riktig dato 😕 Så lenge stringen er på et leselig format, så burde `LocalDate` i backend klare å lage et riktig dato-objekt.